### PR TITLE
hide facet counts - cgap v

### PR DESCRIPTION
**Pull request by Bianca-Morris**
May wait to merge until after Will merges his branch. (~Need to discuss merge/deployment strategy with Will~ merging directly into his branch)

I really only made a couple of changes here (package.json, package.lock to link to SPC branch) and then merged in master (rest, I guess is stuff from master that hasn't been merged into Will's branch yet).

Associated SPC PR: https://github.com/4dn-dcic/shared-portal-components/pull/77

Addressing this ticket: https://trello.com/c/zzuMrRXC/401-add-support-for-no-value-bucket-to-rangefacet

### CODE REVIEW

File: src/encoded/static/components/item-pages/CaseView/InterpretationTab.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ function VariantSampleSelection({ selection, index, caseAccession, schemas }){
                         <div className="flex-grow-1  d-sm-block">
                             &nbsp;
                         </div>
-                        <div className="flex-auto text-small" data-tip="Date Selected">
+                        <div className="flex-auto align-self-center text-small" data-tip="Date Selected">
```
Comment (alexkb0009) [inline src/encoded/static/components/item-pages/CaseView/InterpretationTab.js:104]: (fyi) moved VariantSampleSelection into own/different file in Finalize Case Tab; I've copied over the `link` changes below into it there.

File: src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ export class VariantSampleListController extends React.PureComponent {
         console.info("Fetching VariantSampleList ...");
         const vslFetchCallback = (resp) => {
             console.info("Fetched VariantSampleList", resp);
-            const { "@id": vslID, error = null } = resp;
+            const { 0: { "@id": vslID, error = null } = {} } = resp;
```
Comment (alexkb0009) [inline src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js:87]: Would undo these, have reverted to using datastore=database b.c. of smaller response size.

**Review by alexkb0009 — APPROVED** [review]
Looks good I think... c4_670 should merge master in first maybe?

File: src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js (hunk 2 above)
Comment (Bianca-Morris) [reply src/encoded/static/components/item-pages/CaseView/VariantSampleListController.js]: I copied from your file (what's currently pushed works; did a quick test after) and then fixed the comments.

**Pull request merged**